src/openloco/windows/station_list.cpp

```diff
@@ -17,36 +37,738 @@ namespace openloco::ui::windows::station_list
         tab_road_stations,
         tab_airports,
         tab_ship_ports,
-        company_select,
+        company_select, // 10
```

Comment:
```suggestion
        company_select, // 9
```

Reply:
Thanks — I'll just remove this comment entirely.